halyard-proto/halyard-proto.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.google.protobuf'
 
 dependencies {
-  implementation 'com.google.protobuf:protobuf-java:3.0.0'
+  implementation 'com.google.protobuf:protobuf-java'
```

Comment:
For the record, kork specifies the version [here](https://github.com/spinnaker/kork/blob/c11ebc2c3c80fc20645168764239933f2d7546fb/spinnaker-dependencies/spinnaker-dependencies.gradle#L64).